source/Sashimi.Terraform/Sashimi.Terraform.csproj

```diff
@@ -10,14 +10,41 @@
     <Nullable>enable</Nullable>
     <PackageProjectUrl>https://github.com/OctopusDeploy/Sashimi/</PackageProjectUrl>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <NoDefaultExcludes>true</NoDefaultExcludes>
   </PropertyGroup>
   <ItemGroup>
-    <None Include="..\..\artifacts\Calamari.Terraform.zip" LinkBase="tools" Pack="true" PackagePath="tools/" />
     <EmbeddedResource Include="**\*.ps1" />
     <EmbeddedResource Include="**\*.sh" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
I've just made this one look a little better and also fixed the red when opening the csproj files